.github/workflows/build-extension.yml

```diff
@@ -73,7 +73,20 @@ jobs:
         run: echo "branch=${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}" >> $GITHUB_OUTPUT
         id: extract_branch
 
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - run: yarn
+
+      - name: Deploy to Chromatic
+        id: chromatic
+        uses: chromaui/action@latest
+        with:
+          autoAcceptChanges: 'main'
+          projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This deploy action is currently under the `set-download-link`.

Would be better to create a new step entirely (before set-download-link) called `deploy-storybook` with an output of `steps.chromatic.outputs.storybookUrl`. You'll then need to add that step to the `needs` array

Reply:
Ok, thx for feedback. I wasn't sure how to get the id/output to be recognized unless I put it in the step. I had a separate step first, but then the url wasn't correct. I see what you mean here with setting the output.